### ISSUE TITLE
404 error on source on GitHub

### ISSUE THREAD

**Issue by ghost** [Compass/compass#2055]
On Page: http://compass-style.org/reference/compass/css3/border_radius/

I was looking for:
The source code for your Compass border radius mixins

But instead I found:
I have found your link saying Source on Github - but it is a dead link - 404 error.

http://github.com/chriseppstein/compass/blob/stable/frameworks/compass/stylesheets/compass/css3/_border-radius.scss

**Comment by mjhoy** [comment]
yep. this appears to be fixed by https://github.com/Compass/compass/pull/2061

edit... and quite a few other PRs.
